### DISCUSSION TITLE
SMTP server error: MAIL FROM command failed Detail: Only one sender per message, please

### DISCUSSION THREAD

**Discussion by 1boch1** [appwrite/appwrite#5138] [Q&A]
I both tried with sendgrid and mailgun, and as email both gmail and outlook, I followed the official guide and set up the .env, please help, I get this errors:

Email SMTP 
👩‍⚕ Running Appwrite Doctor for version 1.2.0 ...

```
Checking for production best practices...
🔴 Hostname has no public suffix (localhost)
🔴 CNAME target has no public suffix (localhost)
🔴 Not using a unique secret key for encryption
🔴 App environment is set for development
🟢 Abuse protection is enabled
🟢 Console access limits are enabled
🔴 HTTPS force option is disabled
🔴 Logging adapter is disabled

Checking connectivity...
Database............connected 👍
Queue...............connected 👍
Cache...............connected 👍
SMTP.............disconnected 👎
StatsD..............connected 👍
InfluxDB............connected 👍
```

and

```
,503,SMTP server error: MAIL FROM command failed Detail: Only one sender per message, please
appwrite-worker-mails  |  SMTP code: 503 in /usr/src/code/app/workers/mails.php:126
appwrite-worker-mails  | Stack trace:
appwrite-worker-mails  | #0 /usr/src/code/src/Appwrite/Resque/Worker.php(121): MailsV1->run()
appwrite-worker-mails  | #1 /usr/src/code/vendor/resque/php-resque/lib/Resque/Job.php(201): Appwrite\Resque\Worker->perform()
appwrite-worker-mails  | #2 /usr/src/code/vendor/resque/php-resque/lib/Resque/Worker.php(303): Resque_Job->perform()
appwrite-worker-mails  | #3 /usr/src/code/vendor/resque/php-resque/lib/Resque/Worker.php(242): Resque_Worker->perform(Object(Resque_Job))
appwrite-worker-mails  | #4 /usr/src/code/vendor/resque/php-resque/bin/resque(185): Resque_Worker->work('1', false)
appwrite-worker-mails  | #5 {main}
```

**Comment by stnguyen90** [comment]
> ,503,SMTP server error: MAIL FROM command failed Detail: Only one sender per message, please

@1boch1, this error seems to indicate you have some invalid value for the `_APP_SYSTEM_EMAIL_ADDRESS` environment variable. Please run `docker compose exec appwrite-worker-mails vars` and verify the environment variable is correct. It should just be an email like:

```
_APP_SYSTEM_EMAIL_ADDRESS=team@appwrite.io
```

**Reply by 1boch1** [reply]
@stnguyen90  I tried, it seems to be ok, actually I get this error much more: 

**smtp error: data not accepted. in /usr/src/code/app/workers/mails.php:68**

I tried to set the _APP_SYSTEM_EMAIL_ADDRESS in a lot of different ways, but it gives the same error.
If I try to do "reset password" sometimes it works and I receive the email (still not every time), but with the create email verification from my Vue3.js code it never worked.

As I said I tried both with sandgrid and mailgun, I created an association with the email of my gmail account and grabbed the SMTP settings to be used in my .env file (I reset the docker after)

**Reply by stnguyen90** [reply]
> smtp error: data not accepted

This is an error from your SMTP provider saying some value supplied was invalid. For example, with mailgun, the recipient needs to be added as an authorized recipient or perhaps the `_APP_SYSTEM_EMAIL_ADDRESS` doesn't match the `_APP_SMTP_USER`.

Did you run `docker compose up -d` to apply changes to your .env file? Does `docker compose exec appwrite-worker-mails vars` show the correct values?
